README.md

```diff
@@ -38,34 +38,20 @@ multipass help <command>
 
 Here's a set of steps to build and run your own build of multipass:
 
-## Build Dependencies
+## Install snapcraft
 
 ```
-cd <multipass>
-apt install devscripts equivs
-mk-build-deps -s sudo -i
+sudo snap install snapcraft
```

Comment:
--classic needed

Reply:
after I install snapcraft, my $PATH is not updated to add /snap/bin, so the next snapcraft command fails unless I know about that/open new shell